ui/shared/DropdownMenuItems.jsx

```diff
@@ -31,16 +31,14 @@ const DropdownMenuItems = ({
   <DropdownMenu className="overflow-auto dropdown-menu-height">
     {/* Items pinned to top of dropdown */}
     {pinned.length > 0 &&
-      options
-        .filter(item => pinned.includes(item))
-        .sort((a, b) => a > b)
+      pinned
+        .filter(item => options.includes(item))
```

Comment:
We've already performed this filtering in the client code, so this step is redundant.